spec/controllers/stories_controller_spec.rb

```diff
@@ -0,0 +1,141 @@
+require 'rails_helper'
```

Comment:
In our project, RSpec tests go in tests/spec. Please move all the models, controllers etc specs into that folder. (same structure, just we put it in a slightly different place than normal - I want all the specs (regardless of language) together-ish...